### DISCUSSION TITLE
Help in interpreting message

### DISCUSSION THREAD

**Discussion by thistlillo** [microsoft/vscode-jupyter#11205] [Questions and Answers]
I had problem in opening a .ipynb file in a new remote SSH environment set up on a linux machine using conda. When I ran a cell, VSC used to get stuck at "connecting to kernel", so I disabled the Jupyter extension.

Now I would like to re-enable the Jupyter extension, but I get this message:
```
Cannot change the enablement of Jupyter Notebook Renderers extension because of its extension kind
```

So: what does that mean?
I have continuously new problems with remote notebook development with VSC but I have never encountered this message, that I am not able decipher. 

BTW: Why do VSC python exts need to use error messages of the same incredibly low quality as in python? You could use better messages and still offer python extensions :-)

Using:
```
Version: 1.68.1 (user setup)
Commit: 30d9c6cd9483b2cc586687151bcbcd635f373630
Date: 2022-06-14T12:48:58.283Z
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.19043
```

target is linux ubuntu 20.04.4 LTS (Focal Fossa)

Jupyter extension is version `v2022.5.1001601848`
Python extension is `v2022.12.1`

**Comment by rebornix** [comment]
@sandy081 can you help with this one, not sure if users are seeing the correct error message.

**Comment by roblourens** [comment]
@thistlillo instead of trying to reenable the "Jupyter Notebook Renderers" extension, could you try the extension that is just called "Jupyter"?

**Reply by thistlillo** [reply]
Hello, I got that message when I tried to re-enable the extension Jupyter.

I uninstalled and re-installed everything, now it runs fine.

**Comment by sandy081** [comment]
@thistlillo It seems you are on a bit older version of VS Code. Can you please update to latest and try again?
